uber/site_sections/summary.py

```diff
@@ -1,6 +1,6 @@
 from uber.common import *
 
-@all_renderable(PEOPLE, STATS)
```

Comment:
Agree, recommend we either back this out for master or go ahead and implement as a config option.



> On Nov 27, 2014, at 12:11 AM, Victoria Earl <notifications@github.com> wrote:
> 
> In uber/site_sections/summary.py:
> 
> > @@ -1,6 +1,6 @@
> >  from uber.common import *
> >  
> > -@all_renderable(PEOPLE, STATS)
> This is an AC-specific change, because they want to lock stats about badges away from most people.
> 
> —
> Reply to this email directly or view it on GitHub.
>